techniques.json

```diff
@@ -0,0 +1,183 @@
+[
+    {
+    "type" : "technique",
+        "id" : "HOOK",
+        "name" : "hOOK",

```

Comment:
All the "name" fields have the first letter in lowercase, and the rest in uppercase, is that intentional?
(lines 5, 24, 50, 76, 99, 123, 146, 164)

Reply:
Thank you for bringing that to my attention, I made new files and fixed it.

It was to help me find them because I used to have them all merged into one messy file with the original techniques.json.

Reply:
What directory do I change to, and how do I go about doing that? Also, is the mod file supposed to be in a zip file?